src/javascripts/application.js

```diff
@@ -5,6 +5,7 @@ import Tabs from './components/tabs.js'
 import Copy from './components/copy.js'
 import MobileNav from './components/mobile-navigation.js'
 import Search from './components/search.js'
+import OptionsTable from './components/options-table.js'
```

Comment:
this needs to also be replicated in application-ie8.js as well

Reply:
Great shout, thanks @igloosi. Fixed now.